packages/core/src/index.ts

```diff
@@ -21,7 +21,6 @@ import type {ParsedLayer} from './determine-layer-types'
 import type {BoardShape, BoardShapeRender} from './board-shape'
 import type {Side, SideLayers} from './sort-layers'
 
-export {stringifySvg} from './stringify-svg'
```

Comment:
This is an **export**, not an import. It is a required export of this package and should not be removed